integration-tests/application-server-integration-tests/src/test/java/co/elastic/apm/servlet/WebLogicIT.java

```diff
@@ -33,7 +33,6 @@
 import java.util.Arrays;
 import java.util.List;
 
-@Ignore("Requires docker login which is not available on the CI performing the releases")
```

Comment:
I think this needs to be from [Oracle's registry](https://container-registry.oracle.com/ords/f?p=113:4:29178496937377:::4:P4_REPOSITORY,AI_REPOSITORY,AI_REPOSITORY_NAME,P4_REPOSITORY_NAME,P4_EULA_ID,P4_BUSINESS_AREA_ID:5,5,Oracle%20WebLogic%20Server,Oracle%20WebLogic%20Server,1,0&cs=3z3KVa1Y8Abx01V36GcIyV8JBZQAZYM3RkI0uKIjDUYB0x6NX2XqsViToiSTbR0CnEKNl9NNKzUIgOMIyU7wlDA), see also 
https://oracle.github.io/weblogic-kubernetes-operator/2.6/userguide/managing-domains/domain-in-image/base-images/#obtaining-standard-images-from-the-oracle-container-registry

If we're not doing this now, can we make this draft?